source/main/gui/panels/GUI_GameSettings.cpp

```diff
@@ -251,7 +251,7 @@ void GameSettings::Draw()
     {
         ImGui::TextDisabled(_LC("GameSettings", "Video settings"));
 
-        DrawGCombo(App::gfx_flares_mode, _LC("GameSettings", "Lights"),
+        DrawGCombo(App::gfx_flares_mode, _LC("GameSettings", "Lights (requires restart)"),
```

Comment:
Not true. I tried returning to menu, setting lights to NONE and turning on headlights - the HUD indicator lit up, but the flares didn't appear.

Reply:
If you don't restart RoR, it will crash (try the ant12 plane)